### PR TITLE
Building urls for calls based on attributes

### PR DESCRIPTION
**Pull request by lawong-d2l**

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
       <h3>Basic activity-exemptions demo</h3>
       <demo-snippet>
         <template>
-          <activity-exemptions test-data></activity-exemptions>
+          <activity-exemptions test-data host-url="" org-unit-id activity-id=""></activity-exemptions>
```
Comment (scasagrande-d2l) [inline demo/index.html]: With `org-unit-id` defined as so, does this make `orgUnitId === 1` ?

File: activity-exemptions.html [hunk 2]
```diff
@@ -167,7 +190,7 @@
 			mapUserData: function() {
 				const userData = this.rawClasslist.map( (user) => {
 					const isExempt = this.rawExemptions.some( (exemptions) => {
-						return exemptions['UserId'] === user.Identifier;
+						return exemptions['UserId'] === Number(user.Identifier);
```
Comment (lawong-d2l) [inline activity-exemptions.html]: Classlist and exemptions api returns userid in different formats. Hence this cast.

**Comment by RyanSweetD2L** [comment]
:shipit:

File: activity-exemptions.html [hunk 3]
```diff
@@ -157,6 +172,14 @@
 				this.toMap(e.detail.response);
 			},
 
+			_getClasslistUrl: function(hostUrl, orgUnitId) {
+				return `https://${hostUrl}/d2l/api/le/1.12/${orgUnitId}/classlist/`;
+			},
+
+			_getExemptionsUrl: function(hostUrl, orgUnitId, activityId) {
+				return `https://${hostUrl}/d2l/api/le/1.26/${orgUnitId}/activities/exemptions/?userId=&activityId=${activityId}&showHistory=false`;
```
Comment (jwong-d2l) [inline activity-exemptions.html]: we should default showHistory to false, if we haven't already.

Comment (lawong-d2l) [reply activity-exemptions.html]: Did you want it removed from this url?

File: activity-exemptions.html [hunk 4]
```diff
@@ -43,7 +43,8 @@
 
 		<iron-ajax
 		id="exemptionsRequest"
-		url=""
+		url="[[exemptionsUrl]]"
+		headers='{ "Authorization": "Bearer "}'
```
Comment (jbydeley-d2l) [inline activity-exemptions.html]: These headers aren't required. We're throwing it directly in the LMS.

Comment (lawong-d2l) [reply activity-exemptions.html]: Yah these are here right now for dev/testing. Ill take them out.

**Pull request merged**